src/com/walmartlabs/system_viz.clj

```diff
@@ -109,21 +109,21 @@
   :decorator
   : A function that accepts a component key and component map, and
     returns nil, or a map of additional attributes.
-    
+
   :save-as
-  : If provided, then this is the path to which the graphviz source file 
+  : If provided, then this is the path to which the graphviz source file
     will be saved.  If not provided, the graphviz source file is generated
     as a temporary file.
-      
+
```

Comment:
:save-image-as is not documented.  Also, I would call it :output-to -- but what do I know, :save-as now looks like a pretty crappy name.

Reply:
Yeah i think i considered renaming save-as but didn't want to break backwards compat. What about 'output-image-to'? I was trying to differentiate the graphiz source file from the image file i think.